src/matteruser.js

```diff
@@ -409,7 +450,7 @@ class Matteruser extends Adapter {
 
   userRemoved(msg) {
     // update channels when this bot is removed from a channel
-    if (msg.broadcast.user_id === this.self.id) {
+    if (msg.data.user_id === this.self.id) {
```

Comment:
I suppose it was a typo. userAdded method test `msg.data.user_id === this.self.id` and `getUserByID` use `msg.data.user_id`